src/etc/cargo.bashcomp.sh

```diff
@@ -120,7 +120,7 @@ _cargo()
 } &&
 complete -F _cargo cargo
 
-__cargo_commands=$(cargo --list 2>/dev/null | tail -n +2)
+__cargo_commands=$(cargo --list 2>/dev/null | tail -n +2 | sed -E -e "s/^\s+//g" | cut -d" " -f1)
```

Comment:
This doesn't quite work on BSD-style systems like MacOS due to the \s.  I would use awk which I find to be a little more straightforward.

```suggestion
__cargo_commands=$(cargo --list 2>/dev/null | awk 'NR>1 {print $1}')
```